React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -30,7 +30,11 @@ @interface RCTTextInputComponentView () <RCTBackedTextInputDelegate, RCTTextInpu
 
 @implementation RCTTextInputComponentView {
   TextInputShadowNode::ConcreteState::Shared _state;
-  RCTUIView<RCTBackedTextInputViewProtocol> *_backedTextInputView; // [macOS]
+#if !TARGET_OS_OSX // [macOS]
+  RCTUIView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
+#else // [macOS
+  RCTUITextView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
+#endif // macOS]
```

Comment:
This feels like a significant change from Core?

Reply:
We plan to spend a good chunk of time on TextInput due to the complexity. If we can shim this for the short-term & then revisit once 0.71 lands and we properly adapt for Fabric.